src/sentry/utils/snuba.py

```diff
@@ -1025,6 +1025,14 @@ def _bulk_snuba_query(
                 raise SnubaError("Failed to parse snuba error response")
             raise UnexpectedResponseError(f"Could not decode JSON response: {response.data!r}")
 
+        if "quota_allowance" in body and body["quota_allowance"]:
+            quota_allowance_summary = body["quota_allowance"]["summary"]
```

Comment:
Looks like here if "quota_allowance" exists it doesn't necessary mean that the "summary" exists as well. At least not in the test.

`File "/Users/athena1/code/sentry/src/sentry/utils/snuba.py", line 1036, in _bulk_snuba_query
    quota_allowance_summary = body["quota_allowance"]["summary"]
                              ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 'summary'`